doc/tokens/tokentypes/questionnaire.rst

```diff
@@ -8,15 +8,21 @@ Questionnaire Token
 The administrator can define a list of questions and also how many answers to
 the questions a user needs to define.
 
-During enrollment of such a *question* type token the user answers at least as
-many questions as specified with answers only he knows.
+During enrollment of such a *questionnaire* type token, the user answers at least as
+many questions as specified by the administrator with answers only he knows.
+
+.. figure:: images/enroll_questionnaire.png
+   :width: 500
 
 This token is a challenge response token.
-During authentication the user must give the token PIN and the a random
+During authentication the user gives the token PIN and a random
```

Comment:
During authentication the user first gives the token PIN and the he is presented with a random question to which he gives the answer only he knows.

Reply:
done. I added a note about the change of the aswers of existing tokens.